emoji/src/main/java/chat/rocket/android/emoji/EmojiPickerPopup.kt

```diff
@@ -27,8 +29,10 @@ class EmojiPickerPopup(context: Context) : Dialog(context) {
         setContentView(R.layout.emoji_picker)
 
         tabs.setupWithViewPager(pager_categories)
-        setupViewPager()
-        setSize()
+        launch(UI) {
```

Comment:
If this is a Fragment, you can use `ui { }`

Reply:
This extension is on the app module.